diff -Nur diff/src/diff.c diff_new/src/diff.c
--- diff/src/diff.c	2017-05-06 21:02:54.000000000 +0200
+++ diff_new/src/diff.c	2018-03-12 22:40:07.487423401 +0100
@@ -805,7 +805,7 @@
   print_message_queue ();
 
   check_stdout ();
-  exit (exit_status);
+  //exit (exit_status);
   return exit_status;
 }
 
@@ -883,8 +883,8 @@
 {
   if (ferror (stdout))
     fatal ("write failed");
-  else if (fclose (stdout) != 0)
-    pfatal_with_name (_("standard output"));
+//  else if (fclose (stdout) != 0)
+//    pfatal_with_name (_("standard output"));
 }
 
 static char const * const option_help_msgid[] = {
